src/stories/20_Features/Animations.html

```diff
@@ -212,6 +215,50 @@ <h1>whoosh-in-right</h1>
         </div>
       </amp-story-grid-layer>
     </amp-story-page>
+
+    <!-- ## Ken Burns Effect -->
```

Comment:
You don't need to put every line into a separate comment, something like this should work:

```
<!--- 
## Ken Burns

You can achieve a "Ken Burns" effect by combining the 'pan' and 'zoom' animations. Here's how to do it:

* ...
* ...

-->